### DISCUSSION TITLE
How long does it usually take for the newest version on Github to be released on Google?

### DISCUSSION THREAD

**Discussion by TeamLinux01** [jellyfin/jellyfin-androidtv#4360] [General]
I noticed that 0.18.3 for AndroidTV was released a day or so ago, but 0.18.2 is still on the Google Play store.

The reason I ask is that the repeat bug is currently affecting my use and it would be nice to know when that will be fixed on the Play store version. If it is going to be a few more days still, I might just uninstall and manually install the Github apk release.

**Comment by Dnkhatri** [comment]
No need to uninstall you can just install from GitHub it will preserve your settings. From what I recall update schedule is set at 10% a day so takes 10 days for the release to get to everyone. Gives chance for the dev to stop the release if a major breaking bug is found

**Comment by TeamLinux01** [comment]
Thank you for the response. The staggered release makes sense.

The reason I mentioned an uninstall was that when I tried to installed the APK on another Chromecast with the Google Play version installed, it had an error installing until I uninstalled the Google Play version. This was not on my main Chromecast. It did fix my issue with the repeat function, so I am excited for the update.
